xml/System.Threading.Channels/BoundedChannelOptions.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Provides options that control the behavior of <see cref="T:System.Threading.Channels.BoundedChannel`1" /> instances.</summary>
+    <summary>Provides options that control the behavior of bounded <see cref="T:System.Threading.Channels.Channel`1" /> instances.</summary>
```

Comment:
@stephentoub since this came straight from the /// comments, can you confirm this is the right change?

Reply:
Yup, that's fine, thanks.